Create ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md (#1840)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1019-13-3-ng-cli-prompt-verison-limit-is-now-5-additionally-the-latest.md

---
date: ""

> title: The version selection is more clear when scaffolding a new application with Angular CLI

Reviewer: ```suggestion
title: Clearer version selection when scaffolding a new application with Angular CLI
```

> title: Ng cli prompt verison limit is now 5. Additionally, the latest tag is now shown for the correct version.

Author: The Claude AI suggested to use the following title:
```suggestion
title: ng-cli prompt version limit increased to 5 and latest tag now shown for correct version
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57339
version: 1019.13.3
---

> Ng cli prompt verison limit is now 5. Additionally, the latest tag is now shown for the correct version.

Author: The Claude AI suggested to use the following body:
```suggestion
The ng-cli prompt, which is used to create and manage Angular projects, previously had a version limit of 3. This has now been increased to 5, allowing users to select from a wider range of Angular versions when creating a new project. Additionally, a bug has been fixed where the "latest" tag was not always shown for the correct Angular version in the prompt. With this change, users will now see the "latest" tag next to the most recent stable version of Angular, making it easier to select the desired version when creating a new project. These changes improve the user experience and flexibility when working with the ng-cli prompt.
```

> The Angular CLI new command, which is used to create and manage Web SDK based Angular projects, previously had a version limit of 3. This has now been increased to 5, allowing users to select from a wider range of Web SDK versions when creating a new project. Additionally, a bug has been fixed where the "latest" tag was not always shown for the correct Web SDK version in the prompt. With this change, users will now see the "latest" tag next to the most recent stable version of the Web SDK, making it easier to select the desired version when creating a new project. These changes improve the user experience and flexibility when working with the Angular CLI new comand prompt.

Reviewer: ```suggestion
The Angular CLI new command, which is used to create and manage Web SDK-based Angular projects, previously had a version limit of 3. This has now been increased to 5, allowing users to select from a wider range of Web SDK versions when creating a new project. Additionally, a bug has been fixed where the "latest" tag was not always shown for the correct Web SDK version in the prompt. With this change, users will now see the "latest" tag next to the most recent stable version of the Web SDK, making it easier to select the desired version when creating a new project. These changes improve the user experience and flexibility when working with the Angular CLI new command prompt.
```
